### DISCUSSION TITLE
posts/how-to-copy-folder/

### DISCUSSION THREAD

**Discussion by giscus** [BYJRK/byjrk.github.io#23] [Announcements]
# posts/how-to-copy-folder/

本文介绍了三种在 C# 中拷贝文件夹的方法，分别是使用递归、不使用递归、以及使用 VisualBasic 的内置方法。这三种方法各有优劣，读者可以根据自己的需求来选择适合的方法。

https://blog.coldwind.top/posts/how-to-copy-folder/

**Comment by Captain-Tom-CL** [comment]
好!

**Comment by Anawaert** [comment]
一直感觉VB.NET在某些I/O或者是数据处理方面比C#是强的，比如表达式计算和字符串匹配，不知道是不是以前VB弱类型时代遗传下来的特性😂
